.github/workflows/containers.yml

```diff
@@ -24,6 +25,7 @@ jobs:
         working-directory: docker/network
         run: make ${{ matrix.container }}
       - name: Push to Quay.io
+        if: ${{ env.can_push == true }}
```

Comment:
I wonder, is the `== true` needed?

Reply:
Without this, the if matches. Looks like you must have an expression, and single value is not an expression.